authors/apps/tests/test_articles.py

```diff
@@ -47,7 +47,7 @@ def test_get_all_articles(self):
         self.assertEqual(len(response.json().get("articles")), 3)
 
     def test_get_specific_article(self):
-        slug = self.slugger()
+        slug = self.slugger2()
```

Comment:
If slugger was deleted, why do we do have slugger2()?

Reply:
@malep2007  I apologise for the communication I said earlier, The slugger function was not deleted, we just created another slugger helper method which has a verified user.unlkie the first one.